### PR TITLE
Add 404 checking for graybox-bacom

### PR DESCRIPTION
**Pull request by JackySun9**
Add 404 checking for graybox-bacom

### CODE REVIEW

**Review by amauch-adobe — APPROVED** [review]
Approved.  I would just make sure to have in the .gitignore file the 404 results .json file so that it doesn't get committed to the repo, unless of course you have a reason for committing it.

File: configs/graybox-bacom.config.js [hunk 1]
```diff
@@ -9,7 +9,7 @@ const envs = require('../envs/envs.js');
  */
 const config = {
   testDir: '../tests/',
-  testMatch: ['bacom/**/*.test.js', 'milo/**/*.test.js'],
+  testMatch: ['bacom/**/*.test.js', 'milo/**/*.test.js', 'graybox/**/*.test.js'],
```
Comment (skumar09) [inline configs/graybox-bacom.config.js:12]: @JackySun9 : you want to include milo tests runs also

**Review by skumar09 — APPROVED** [review]

File: configs/graybox-bacom.config.js (hunk 1 above)
Comment (JackySun9) [reply configs/graybox-bacom.config.js:12]: yes, we need to run milo test for graybox webistes

**Comment by JackySun9** [comment]
> Approved. I would just make sure to have in the .gitignore file the 404 results .json file so that it doesn't get committed to the repo, unless of course you have a reason for committing it.

yes, fixed, thanks!

**Pull request merged**